spec/controllers/api/v1/books_controller_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
+
+describe Api::V1::BooksController, type: :controller do
+  include_context 'Authenticated User'
+  describe 'GET #index' do
+    context 'When fetching all the books' do
+      let(:books) { create_list(:book, 3) }
+
+      subject(:http_request) { get :index }
+
+      it 'responses with the books json' do
+        expected = ActiveModel::Serializer::CollectionSerializer.new(
+          books, each_serializer: BookSerializer
+        ).to_json
+
+        expect(subject.body.to_json) =~ JSON.parse(expected)
```

Comment:
```suggestion
        expect(http_request.body.to_json) =~ JSON.parse(expected)
```

Ya que le diste un nombre, usalo!